service/src/test/java/de/adorsys/psd2/sandbox/xs2a/pis/PaymentInitiationWithScaSteps.java

```diff
@@ -39,38 +45,72 @@ public void initiatePayment(String paymentProduct) {
     context.setPaymentProduct(paymentProduct);
 
     HashMap<String, String> headers = TestUtils.createSession();
-    headers.put("psu-ip-address", "192.168.0.26");
 
     PaymentInitiationSctJson payment = new PaymentInitiationSctJson();
     payment.setEndToEndIdentification("WBG-123456789");
 
-    HashMap<String, String> debtorAccount = new HashMap<>();
-    debtorAccount.put("currency", "EUR");
-    debtorAccount.put("iban", "DE51250400903312345678");
+    HashMap<String, String> debtorAccount =  createAccount("DE51250400903312345678");
     payment.setDebtorAccount(debtorAccount);
 
     Amount instructedAmount = new Amount();
     instructedAmount.setAmount("520");
     instructedAmount.setCurrency("EUR");
     payment.setInstructedAmount(instructedAmount);
 
-    HashMap<String, String> creditorAccount = new HashMap<>();
-    creditorAccount.put("currency", "EUR");
-    creditorAccount.put("iban", "DE15500105172295759744");
+    HashMap<String, String> creditorAccount = createAccount("DE15500105172295759744");
```

Comment:
`createAccount("")` can be used inline assigning it to `HashMap<String, String> creditorAccount` first

Reply:
done